backend/util/tracer.go

```diff
@@ -52,9 +54,9 @@ func StartSpanFromContext(ctx context.Context, operationName string, options ...
 		opt(&cfg)
 	}
 
-	hTracingDisabled, _ := ctx.Value(ContextKeyHighlightTracingDisabled).(bool)
+	hTracingDisabled, _ := ctx.Value(highlightTracingDisabled).(bool)
 	hTracingDisabled = hTracingDisabled || cfg.HighlightTracingDisabled
-	ctx = context.WithValue(ctx, ContextKeyHighlightTracingDisabled, hTracingDisabled)
+	ctx = context.WithValue(ctx, highlightTracingDisabled, hTracingDisabled)
```

Comment:
@Vadman97 is this right to assign an int as the key for the value? Would it be better to use a string?

Reply:
I was getting this lint warning using a string
```
SA1029: should not use built-in type string as key for value; define your own type to avoid collisions (staticcheck)
```
apparently you're supposed to use a key type? https://stackoverflow.com/questions/40891345/fix-should-not-use-basic-type-string-as-key-in-context-withvalue-golint

Reply:
Yeah, I saw that as well and was just looking into it! Was wondering if we should assign a key type of string rather than `iota` though. Our current usage would translate to `WithValue(ctx, 0, hTracingDisabled`. Seems like we could accidentally override that pretty easily with another enum, right?

What do you think about this instead?

```
type contextKey string

const (
	ContextKeyHighlightTracingDisabled contextKey = "HighlightTracingDisabled"
)

func StartSpanFromContext(ctx context.Context, operationName string, options ...SpanOption) (MultiSpan, context.Context) {
	var cfg SpanConfig
	var ddOptions []tracer.StartSpanOption
	for _, opt := range options {
		opt(&cfg)
	}

	hTracingDisabled, _ := ctx.Value(string(ContextKeyHighlightTracingDisabled)).(bool)
	hTracingDisabled = hTracingDisabled || cfg.HighlightTracingDisabled
	ctx = context.WithValue(ctx, ContextKeyHighlightTracingDisabled, hTracingDisabled)

...
```

Reply:
ooo my mistake! yeah that sounds better, feel free to make that change